jobs/payment-jobs/tasks/ejv_payment_task.py

```diff
@@ -157,10 +182,15 @@ def _create_ejv_file_for_gov_account(cls, batch_type: str):  # pylint:disable=to
                             effective_date,
                             flow_through,
                             journal_name,
-                            line.total,
+                            line_total,
                             line_number,
-                            "D" if not is_jv_reversal else "C",
+                            "D" if not is_jv_reversal or not is_partial_refund else "C",
                         )
+                        # If it's partial refund, mark the GL posted as True, so that it's not processed again
+                        if is_partial_refund:
+                            matching_refund.gl_posted = datetime.now(tz=timezone.utc)
+                            # skip the service fee cause we never partially refund service fees
```

Comment:
Any thoughts on service_fee partial_refund disbursement? I asked IT OPS, we're not doing any service partial_refund currently.

Reply:
As long as that's a limitation in the GUI, I think currently the GUI you can just refund the service fees (at least for credit card)?

Reply:
ok I'll add for it

Reply:
@seeker25 current ejv_payment_task not handle full refund of priority fee and future_effective_fee, only service_fee